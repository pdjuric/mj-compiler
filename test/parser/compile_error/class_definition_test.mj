program main_test

class Ok {}
Ok ok;
const int c_ok = 5;

class char {}
class Ok {}
class ok {}
class c_ok {}
class ord {}
class len {}

{
    void main(){}
}