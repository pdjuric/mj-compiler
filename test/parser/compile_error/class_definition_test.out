ERROR 14:12:50,479 - line   8 - Identifier Ok already defined in this scope.
ERROR 14:12:50,479 - line   9 - Identifier ok already defined in this scope.
ERROR 14:12:50,479 - line  10 - Identifier c_ok already defined in this scope.